docs/content/docs/changelogs/overview.md

```diff
@@ -1,6 +1,8 @@
 ---
-title: Releases and changelogs
+title: Overview
```

Comment:
Is the plan to have all the releases in the sidebar (like in the site preview here)?  If so I don't think this page serves much purpose.  (Though I also don't think having them all on the sidebar is good because of how many releases there are going to be

Reply:
I think for right now, this is the best place, but once there are a lot we should move them to their own nav, like VSCode does. I have a local change I'm working on for that but I don't think it's blocking.

Reply:
Sounds good.  But until then, I still don't think this page is very useful as is

Reply:
Agreed. Removed.